addons/api/mirage/scenarios/ipc.js

```diff
@@ -139,7 +132,7 @@ export default function initializeMockIPC(server, config) {
      */
     minimizeWindow() {}
     closeWindow() {}
-    toggleFullScreenWindow() {}
+    toggleFullscreenWindow() {}
```

Comment:
There was a typo here and we weren't actually correctly mocking this one

Reply:
Good catch!